HealthcareBase/Migrations/20201105000902_DataSeed.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthcareBase.Migrations
+{
+    public partial class DataSeed : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```

Comment:
Is the data entered in the method "Up" only test/dummy data and did they remain there by mistake? 
@sStrbacki

Reply:
Data is seeded only for testing purposes. 
Dummy data is populated in `OnModelCreating(ModelBuilder modelBuilder)` method.
For more info see [Data Seeding - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/data-seeding)

Reply:
It is clear that it is an extremely useful thing, but master shouldn't to contain dummy data.

Reply:
I agree, master branch shouldn't contain dummy data, but at this very moment, I think that this way of populating  test data is extremely handy. Instead of writing sql queries by hand, others can use DataSeed migration in order to populate sql tables.

Reply:
Since this is the beginning of the project and this will speed up the work on it, we will currently allow it.
It will be necessary to remove the dummy data from the master and solve the problem in another way